fcomm/examples/function.json

```diff
@@ -1 +1 @@
-{"Source":"(lambda (a) (+ 7 (* a a)))","secret":[11058593017581833556,17028083709131530449,17225780588865781989,5089294428800148454],"commitment":"3e3beecca5b9d817646aaa0f48e7bc92193e6fbf8724ee77a378c0ea91988b20"}
+{"fun":{"Source":"(lambda (a) (+ 7 (* a a)))"},"secret":[11058593017581833556,17028083709131530449,17225780588865781989,5089294428800148454],"commitment":"3e3beecca5b9d817646aaa0f48e7bc92193e6fbf8724ee77a378c0ea91988b20"}
```

Comment:
minor one: why are we using uppercase `S` on the source? can we just use lowercase `s`? 

I assume fun here is "function", maybe it's better to use `func`?

Reply:
The uppercase is an artifact of the automatic serialization we get when using a Rust struct. The shape of these data structures is still a little ad-hoc and walks a tightrope of multiple requirements. It will probably continue to evolve, so let's not get too attached to small details yet.

There is a Lurk type called `Fun` (which represents a function), and the field here is meant to specify a value of that type. (When specified by source, it will need to be evaluated to create a real `Fun`, but we can also specify IPLD that maps directly to the `Fun` itself.)